src/convention/coerce.jl

```diff
@@ -168,3 +168,36 @@ function _check_tight(v::CArr, T, tight)
     end
     return v
 end
+
+
+## Images
+F = Union{Float32, FixedPointNumbers.Normed{UInt8,8}}
+_2d{C} =AbstractArray{C} where C<:Union{ColorTypes.AbstractRGB, ColorTypes.Gray}
+
+# Single Image
+function coerce(y::Arr{<:F, 2}, T2::Type{_2d{ColorTypes.Gray}})
+    return ColorTypes.Gray.(y)
+end
+
+function coerce(y::Arr{<:F, 3}, T2::Type{_2d{ColorTypes.AbstractRGB}})
+    return ColorTypes.RGB.(y)
+end
+
```

Comment:
This one should convert a 3d array into a 2d array but doesn't appear to. Shouldn't it be more like this?

```julia
function coerce(y::Arr{<:F, 3}, T2::Type{ColorImage})
    return broadcast(ColorTypes.RGB, y[:,:,1], y[:,:,2], y[:,:,3])
end
```